grabbit/src/main/groovy/com/twcable/grabbit/client/servlets/GrabbitRootServlet.groovy

```diff
@@ -0,0 +1,47 @@
+package com.twcable.grabbit.client.servlets
+
+import groovy.transform.CompileStatic
+import groovy.util.logging.Slf4j
+import org.apache.felix.scr.annotations.sling.SlingServlet
+import org.apache.sling.api.SlingHttpServletRequest
+import org.apache.sling.api.SlingHttpServletResponse
+import org.apache.sling.api.servlets.SlingAllMethodsServlet
+
+/**
+ * Copyright 2015 Time Warner Cable, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Root resource for /grabbit requests.
+ * Acts as a handler for {@link com.twcable.grabbit.resources.RootResource} resource.
+ */
+@Slf4j
+@CompileStatic
+@SlingServlet(methods = ['GET', 'PUT'], resourceTypes = ['twcable:grabbit'])
+class GrabbitRootServlet extends  SlingAllMethodsServlet {
+
+    /**
+     * Response with links to other request urls i.e. job, content or transaction requests in HTML format
+     */
+    @Override
+    void doGet(SlingHttpServletRequest request, SlingHttpServletResponse response) {
+        response.setContentType("text/html");
```

Comment:
Hmm, what about other content types? Should we force html like this? Should we return a 200 in this case, or does a 300 Multiple Choice make sense?